bot/exts/moderation/incidents.py

```diff
@@ -291,6 +403,9 @@ async def process_event(self, reaction: str, incident: discord.Message, member:
         This ensures that if there is a racing event awaiting the lock, it will fail to find the
         message, and will abort. There is a `timeout` to ensure that this doesn't hold the lock
         forever should something go wrong.
+
+        Deletes cache value (`message_link_embeds_cache`) of `msg_before` if it exists and removes the
```

Comment:
Got a bit confused by the wording at first
```suggestion
        Deletes cache value (`message_link_embeds_cache`) of `msg_before` if it exists. It then removes the
```
Also I don't see any variable called `msg_before`. It will also be helpful if you mention what that variable represents in your own words, since people will be first reading the docstring and then the code.